test/tests/rebar/truss_inelastic.i

```diff
@@ -0,0 +1,126 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  elem_type = EDGE
+  nx = 1
+  displacements = 'disp_x'
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[AuxVariables]
+ [./axial_stress]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./e_over_l]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./area]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./react_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[BCs]
+  [./fixx1]
+    type = DirichletBC
+    variable = disp_x
+    boundary = left
+    value = 0.0
+  [../]
+  [./load]
+    type = FunctionPresetBC
+    variable = disp_x
+    boundary = right
+    function = '1e-2*t'
+  [../]
+[]
+
+[AuxKernels]
+  [./axial_stress]
+    type = MaterialRealAux
+    property = axial_stress
+    variable = axial_stress
+  [../]
+  [./e_over_l]
+    type = MaterialRealAux
+    property = e_over_l
+    variable = e_over_l
+  [../]
+  [./area]
+    type = ConstantAux
+    variable = area
+    value = 1.0
+    execute_on = 'initial timestep_begin'
+  [../]
+[]
+
+[Postprocessors]
+  [./s_xx]
+    type = ElementIntegralMaterialProperty
+    mat_prop = axial_stress
+  [../]
+  [./e_xx]
+    type = ElementIntegralMaterialProperty
+    mat_prop = total_stretch
+  [../]
+  [./ee_xx]
+    type = ElementIntegralMaterialProperty
+    mat_prop = elastic_stretch
+  [../]
+  [./ep_xx]
+    type = ElementIntegralMaterialProperty
+    mat_prop = plastic_stretch
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+  petsc_options = '-snes_ksp_ew'
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
+  nl_abs_tol = 1e-11
+  l_max_its = 20
+  dt = 1.0
+  num_steps = 3
+[]
+
+[Kernels]
+  [./solid]
+    type = StressDivergenceTensorsTruss
+    displacements = 'disp_x'
```

Comment:
Put 'displacements' in GlobalParams and remove it in the 4 places it shows up in various models.